### ISSUE TITLE
Map a network drive flyout button blocked by Drives

### ISSUE THREAD

**Issue by BehroozRezvani** [files-community/Files#11079]
### Description

The flyout button for map a network drive inside the Drives widget on the home screen can be blocked by the Drives if the window resized in a specific size, and therefore not clickable (even though still visible).

### Steps To Reproduce

have at least 2 Drives (e.g. C: and D:) resize the window to the point where flyout button is covered by the drive item. at this point the button is no longer clickable.

### Steps to fix

Change the margin of the ItemsRepeater for the Drives items so that it doesn't clash with flyout button.
Or, move the flyout button somewhere else.

### Files Version

2.4.25

### Windows Version

Windows 11 Pro 22623.891

### Relevant Assets/Logs

![Screenshot_20230125_213738](https://user-images.githubusercontent.com/26872640/214697016-d74df777-3e8b-42f4-b317-90cd6639d396.png)

**Comment by Josh-65** [comment]
For now you can right click the network folder on the sidebar and map a drive from there.

**Comment by yaira2** [comment]
We're tracking this in another issue, I can't find it right now but know that it's on our roadmap to fix.

**Comment by BehroozRezvani** [comment]
I was just about to make a pull request by adding a margin to the ItemsRepeater. :(
But anyway, glad it is getting fixed

**Comment by Josh-65** [comment]
You can still make the PR to fix the issue

**Comment by BehroozRezvani** [comment]
> You can still make the PR to fix the issue

Awesome!

**Comment by BehroozRezvani** [comment]
> The plan is to move the menu to the expander header but some refactoring is needed before that can be done.

Yeah I was expecting that is the case, because the placement of the flyout button seems a bit odd
